### DISCUSSION TITLE
Debian-custom container for Pi-hole stops and does not restart after reboot

### DISCUSSION THREAD

**Discussion by GigaBypass9876** [unifi-utilities/unifi-common#561] [Q&A]
After a UDM SE reboot, I need to manually start the Debian-custom container for Pi-hole. What might be the problem that this stops on a reboot and does not start automatically?
After reboot, when I enter
systemctl status udm-boot
it indicates all is up and running

Then I manually start the container 
machinectl start "debian-custom"
And this gets Pi-hole running until the next reboot or power outage.
Thanks in advance for any help.

**Comment by m4n63** [comment]
I realise this is an old question, but the solution here may be that you have no script to start the container. If you follow the guide you should have a script in the on_boot directory called `0-setup-system.sh` that should start your container on boot.
